src/app/externs/backendapi.js

```diff
@@ -918,12 +917,20 @@ backendApi.LogLine;
 
 /**
  * @typedef {{
- *   referenceLogLineId: !backendApi.LogLineReference,
- *   relativeFrom: number,
- *   relativeTo: number
+ *   referencePoint: !backendApi.LogLineReference,
```

Comment:
The current structure is:

```
LogPage
      LogPageInfo (podname, containername, timestamps )
      LogSelection (reference point to keep track of paging)
      LogLines  (actual logs)
```
LogViewInfo -> LogSelection is syntactically correct. Still confusing?